docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+
+:::warning
+The .net clients prior to version 21.6 had an incorrect behavior: they _will_ save the checkpoint. 
+:::
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Abort
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+// This is currently not directly available in the NodeJS client.
+// However, disableProjection allows to pass writeCheckpoint: false in the options in order to abort the projection  
+await client.disableProjection("projection to abort", { writeCheckpoint: false });
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Trying to abort a projection that does not exists will result in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Abort_NotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+// This is currently not directly available in the NodeJS client.
+// However, disableProjection allows to pass writeCheckpoint: false in the options in order to abort the projection  
+await client.disableProjection("projection to abort", { writeCheckpoint: false });
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Reset a projection 
+Resets a projection. This will re-emit events. Streams that are written to from the projection will also be soft deleted.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Reset
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ resetProjection(projectionName: string,options?: ResetProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Resetting a projection that does not exists will result in an error. 
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Reset_NotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ resetProjection(projectionName: string,options?: ResetProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Create a projection
```

Comment:
**SUGGESTION:** I think that this should be the first snippet, as we should try to have a learning path. If I were looking for those docs then I'd at first want to learn how to create a projection, then how to run it (enable) then other methods like disable, abort and delete.

Reply:
move closer to the top